magento/magento2
# [Backport] Wrong currency on creditmemo_grid & sales_order_view > creditmemo grid

### Description (*)

Added missing column `order_currency_code` to table `sales_creditmemo_grid`. Copying column `order_currency_code` to `sales_creditmemo_grid`. Price column rendering changed on  `sales_creditmemo_index` (Creditmemo grid) & `sales_order_view` - Creditmemo tab.

### Manual testing scenarios (*)

1. Create order in different currency as configured on default store view
2. Go to Backend > SALES > Credit Memos, create Credit Memo
3. Check *Refunded* column on Credit Memo grid. You should see price in currency configured on default store view.
4. Go to Backend > SALES > Orders > choose your order and check Credit Memo grid in Credit Memos tab. You should see price in currency configured on default store view.

sales_order_view:
![image](https://user-images.githubusercontent.com/5650354/55399569-ba6b1980-554b-11e9-9771-255ffb1fe41b.png)

sales_creditmemo_index:
![image](https://user-images.githubusercontent.com/5650354/55399531-a7f0e000-554b-11e9-9bd0-fd935624e105.png)



### Contribution checklist (*)

Backport for: https://github.com/magento/magento2/pull/22123
